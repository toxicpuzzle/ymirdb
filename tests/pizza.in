SET a 99 100 101
SET b 2 56 78
SET c 1 22 16
SET d 1 0 2
SET e 1 2
SET f 23
SET g 101
PUSH a b g
APPEND b g c
PUSH c d g
APPEND d g e
APPEND e f
PUSH g f e
FORWARD a
BACKWARD g
LIST ENTRIES
DEL g
SNAPSHOT
DEL a
SNAPSHOT
DEL b
SNAPSHOT
DEL c
PURGE c
DEL d
PURGE d
ROLLBACK 1
LIST SNAPSHOTS
PURGE a
PURGE b
PURGE c
PURGE d
LIST ENTRIES
BYE
