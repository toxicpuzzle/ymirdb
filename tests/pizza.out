> ok

> ok

> ok

> ok

> ok

> ok

> ok

> ok

> ok

> ok

> ok

> ok

> ok

> b, c, d, e, f, g

> a, b, c, d

> g [e f 101]
f [23]
e [1 2 f]
d [1 0 2 g e]
c [g d 1 22 16]
b [2 56 78 g c]
a [g b 99 100 101]

> not permitted

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> saved as snapshot 3

> ok

> not permitted

> ok

> not permitted

> ok

> 1

> ok

> ok

> ok

> ok

> g [e f 101]
f [23]
e [1 2 f]

> bye
